shared/package.json

```diff
@@ -12,9 +12,11 @@
     "build": "npm run build:ts && cp package.json build"
   },
   "devDependencies": {
+    "@types/amqplib": "^0.8.2",
     "tsc-alias": "1.6.11"
   },
   "dependencies": {
+    "amqplib": "^0.10.2",
```

Comment:
It looks like the amqplib is no needed module in the shared package

Reply:
`shared/src/common/types/amqp/amqp-send-to-queue-dto.type.ts` imports Options from there. that's why #5 resulted into deploy fail, since it wasn't installed in shared/package.json.

Reply:
Could please check if `@types/amqplib` enough for building docker images